contribs/src/main/java/com/netflix/conductor/contribs/lambda/Lambda.java

```diff
@@ -0,0 +1,61 @@
+package com.netflix.conductor.contribs.lambda;
+
+import com.netflix.conductor.common.metadata.tasks.Task;
+import com.netflix.conductor.common.run.Workflow;
+import com.netflix.conductor.core.events.ScriptEvaluator;
+import com.netflix.conductor.core.execution.WorkflowExecutor;
+import com.netflix.conductor.core.execution.tasks.WorkflowSystemTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Singleton;
+import javax.script.ScriptException;
+import java.util.Map;
+
+@Singleton
+public class Lambda extends WorkflowSystemTask {
```

Comment:
@X-Ultra One more nitpick - Can you please rename class to `LambdaTask`, while still keeping the task type as `Lambda`. You can check `HttpTask` for reference. Also, please add javadoc for this class, briefly explaining how this task is supposed to be used. We'll migrate this over to documentation.

Reply:
Okay, it doesn't matter. ~